### PR TITLE
fix(arc-1661): added tooltips to small CTA's

### PR DESCRIPTION
**Pull request by WardVerc1**
image
image
image
image
Ticket: https://meemoo.atlassian.net/browse/ARC-1661?focusedCommentId=40817
⚠️ React-components PR needed: https://github.com/viaacode/react-components/pull/95

### CODE REVIEW

File: src/modules/account/components/EditFolderTitle/EditFolderTitle.tsx [hunk 1]
```diff
@@ -107,19 +110,30 @@ const EditFolderTitle: FC<EditFolderTitleProps> = ({
...
-					variants={['silver']}
-					name={tText(
-						'modules/account/components/edit-folder-title/edit-folder-title___map-aanpassen'
-					)}
-					icon={<Icon name={IconNamesLight.Edit} aria-hidden />}
-					aria-label={tText(
-						'modules/account/components/edit-folder-title/edit-folder-title___titel-aanpassen'
-					)}
-				/>
+				<Tooltip position="top">
```
Comment (silkederudder-shd) [inline src/modules/account/components/EditFolderTitle/EditFolderTitle.tsx:113]: I'd add some extra functionality to the button component so you're able to add the tooltip if wanted. This way, way we don't neet to write the same code over and over again ;)

**Review by silkederudder-shd — CHANGES_REQUESTED** [review]

File: src/modules/account/components/EditFolderTitle/EditFolderTitle.tsx (hunk 1 above)
Comment (bertyhell) [reply src/modules/account/components/EditFolderTitle/EditFolderTitle.tsx:113]: yes or a wrapper: <ButtonWithTooltip>

**Review by bertyhell — CHANGES_REQUESTED** [review]
see remark silke

**Comment by WardVerc1** [comment]
Added tooltip to Button, see: https://github.com/viaacode/react-components/pull/95

**Review by silkederudder-shd — APPROVED** [review]

**Review by bertyhell — APPROVED** [review]

**Pull request merged**